build/conf/kylin-parquet-log4j.properties

```diff
@@ -0,0 +1,31 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#  
+#     http://www.apache.org/licenses/LICENSE-2.0
+#  
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+#overall config
+log4j.rootLogger=WARN,hdfs,stdout
+log4j.logger.org.apache.kylin=DEBUG
+log4j.logger.org.springframework=WARN
+log4j.logger.org.springframework.security=WARN
+log4j.logger.org.apache.spark=WARN
+log4j.logger.org.apache.spark.ContextCleaner=WARN
+
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
```

Comment:
I am not really sure, but I want to ask why not write log into a file?

Reply:
This log4j configuration file is used for Spark job. If the job is submitted to yarn. The normal file appender cannot be used. 
We need to extend a new appender by ourselves to let the log of spark job(both driver and executor) write into HDFS which can be realized in the future.  
Kylin submit a spark job by a command executor client. It will gather the output of spark job and save into kylin.log  .

Reply:
OK